_data/simple-icons.json

```diff
@@ -1845,6 +1845,11 @@
             "hex": "4381C3",
             "source": "https://battle.net"
         },
+        {
+            "title": "Bazel",
+            "hex": "43A047",
+            "source": "https://blog.bazel.build/images/bazel-icon.svg"
```

Comment:
```suggestion
            "source": "https://bazel.build"
```

Reply:
Hi. Could you please post that link, where on a web site you found that icon with cutouts? Because I was referencing their new logo announcement and favicon. I could save SO MUCH time not trying to get those cutouts corectly aligned. (I'm not a designer) :).

Reply:
There isn't one with cutouts, unfortunately. I just used the SVG in the website header, added a 0.25pt outline around _every_ shape, and then just combining the paths to get our usual 0.5pt stroke all around.

Reply:
Ohh... thins means I need to jump into learning brand new design tool for me. I just used Figma and it's capabilities (or my knowledge about it) is bit too limited. Will try to install Inkscape and see how it goes. Thank you.

Reply:
There isn't one with cutouts, unfortunately. I just used the SVG in the website header, added a 0.25pt outline around _every_ shape, and then just combining the paths to get our usual 0.5pt stroke all around.

Reply:
Awesome, drop me a message here or on our Discord if you need any support!

Reply:
All this GitHub PR is really confusing in comparison with other workflows. I amended changes to my original commit. 0.25pt gaps around elements. Should be fine now. Learned new tool a bit. :)